casepro/cases/models.py

```diff
@@ -103,6 +103,16 @@ def __str__(self):
         return self.name
 
 
+class SystemUser(User):
+
+    @classmethod
+    def get_or_create(cls):
+        systemUsers = cls.objects.filter()
+        if len(systemUsers) > 0:
+            return systemUsers[0]
```

Comment:
why not use `.exists()` and `.first()` here? Those have less of a performance hit if there are a lot of users, especially with postgres. Not sure if this is a relevant question as I'm not sure how big the users table is likely to be.